website/blog/2022-07-01-aluminium.md

```diff
@@ -0,0 +1,444 @@
+---
+author: Tomasz Godzik
+title: Metals v0.11.7 - Aluminium
+authorURL: https://twitter.com/<login>
+authorImageURL: https://github.com/<login>.png
+---
+
+We're happy to announce the release of Metals v0.11.7, which brings a large
+number of Scala 3 improvements as well as a few interesting code actions.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">151</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">68</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">12</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">14</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">8</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/51?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs and
+Sublime Text. Metals is developed at the [Scala Center](https://scala.epfl.ch/)
+and [VirtusLab](https://virtuslab.com) with the help from
+[Lunatech](https://lunatech.com) along with contributors from the community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/), and
+give Metals a try!
+
+- Release notes now shown in VS Code
+- Improvements to Bloop Java home automation.
+- Allow to run all scalafix rules on a file
+- [Scala 3] Implement-all abstract members code action.
+- [Scala 3] Add snippet completions.
+- [Scala 3] Add file name completions.
+- [Scala 2] Add ConvertToNamedArguments code action.
+- New flatMap to for comprehension code action.
+- added support for Scala 2.12.16, 3.1.3 and 3.2.0-RC1
```

Comment:
I would ignore scala versions support in release notes as the current release support these versions as well as one that will be release.